ts/Series/BoxPlot/BoxPlotSeries.ts

```diff
@@ -261,7 +261,11 @@ class BoxPlotSeries extends ColumnSeries {
                 let d: SVGPath;
 
                 // The stem
-                const stemX = crisp(point.plotX || 0, point.stem.strokeWidth());
+                const stemX = crisp(
+                    (point.plotX || 0) + (series.pointXOffset || 0) +
```

Comment:
Is there a reason for why we use "||" instead of "??"?
I'd recommend using ?? instead of || as the nullish coalescing operator (??) is regarded as safer and cannot be mistaken as a boolean or operator.

Reply:
Since `point.plotX` is typed as `number|undefined`, I'd say the `||` operator is safe. We use this pattern a lot throughout the code base, for properties of this type.